pydriller/metrics/process/comm.py

```diff
@@ -0,0 +1,30 @@
+from pydriller.repository_mining import RepositoryMining
+
+class COMM():
+    """
+    This class is responsible to implement the process metric "Commit Count"
+    Commit Count measures the number of commits made to a file.
+    """
+
+    def count(self, path_to_repo: str, commit_hash: str, filename: str):
```

Comment:
What I was thinking is to calculate all the metrics inside this class. Hence, this function would become `commit_count`. 
And later we will have `active_dev_count`, `distinct_dev_count`, etc.

Reply:
`filename` should probably be `filepath` (otherwise it fails for 2 files with the same filename).

Reply:
I would also put `commit_hash` as optional, in the case in which the researcher wants to analyse the entire history of the project.

Reply:
I agree on everything.

Reply:
A possible improvement (maybe for the future?) would be to put the filepath as optional. If this is the case, then we would calculate the specific metric for **all** the files.

Reply:
I think it will make the code a bit redundant and less self-contained. I would keep going with the single filepath (one can call the function for each file of interest). Let's think with calm to that idea.